store/tikv/coprocessor.go

```diff
@@ -168,7 +291,7 @@ func (it *copIterator) Next() (io.ReadCloser, error) {
 		resp, err := client.SendCopReq(req)
 		if err != nil {
```

Comment:
This error should be logged before `it.rebuildCurrentTask`, the error value in the following log would always be nil.